test/libsolidity/InlineAssembly.cpp

```diff
@@ -429,6 +429,16 @@ BOOST_AUTO_TEST_CASE(revert)
 	BOOST_CHECK(successAssemble("{ revert(0, 0) }"));
 }
 
+BOOST_AUTO_TEST_CASE(returndatasize)
+{
+	BOOST_CHECK(successAssemble("{ let r := returndatasize }"));
```

Comment:
`add` is already tested for both styles in `functional`, but it has two arguments: not zero or three.